vespalib/src/vespa/vespalib/stllike/hashtable.h

```diff
@@ -103,26 +103,68 @@ class hash_node {
     enum {npos=-1u, invalid=-2u};
     hash_node() : _node(), _next(invalid) {}
     hash_node(const V & node, next_t next=npos)
-        : _node(node), _next(next) {}
+        : _node(),
```

Comment:
I think we can skip 'node' double-initialization by not initializing 'node' here, not having to rely too heavily on compiler optimizations.

Reply:
Fixed